### ISSUE TITLE
`Mesh` : check case `mesh` was provided with no `.material` or `.geometry` set.

### ISSUE THREAD

**Issue by vatro** [vatro/svelthree#84]
`"this will most probably never happen"` or `"it can / will never happen"` ?
If in both cases three.js throws an Error (_I'm pretty sure it will_) at some point, we don't need to do it -> simplify.

https://github.com/vatro/svelthree/blob/8fb54d28b52b43532b15600bd01d4712f2032ed6/src/components/Mesh.svelte#L121-L148

**Comment by vatro** [comment]
```js

// NO ERROR
// has default geometry: empty `BufferGeometry`
// has default material: white `BasicMaterial`
const new_mesh = new _Mesh()

// three.js throws Error: `Uncaught (in promise) TypeError: Cannot read properties of null (reading 'morphAttributes')`
const new_mesh = new _Mesh(null, null)

// SVELTHREE throws Error: `Error: SVELTHREE > Mesh : 'mesh' provided, but has no geometry!`
const new_mesh = new _Mesh()
new_mesh.material = null
new_mesh.geometry = null

// SVELTHREE throws Error: `Error: SVELTHREE > Mesh : 'mesh' provided, but has no material!`
const new_mesh = new _Mesh()
new_mesh.material = null

// SVELTHREE throws Error: Error: `SVELTHREE > Mesh Error: provided 'mesh' instance has wrong type 'OrthographicCamera', should be 'Mesh'!`
const new_mesh = new _OrthographicCamera()
```

Means: it's ok as it is, but remove unsure comments.

**Comment by vatro** [comment]
resolved, see 3478ee6ff0266af9696035ec9a2d54edc3ac6676
